examples/serverless-batch.yaml

```diff
@@ -35,28 +35,23 @@ deployment_groups:
     use: [network1]
     settings: {local_mount: /sw}
 
-  - id: load-data-startup
-    source: modules/scripts/startup-script
-    settings:
-      runners:
-      - type: shell
-        destination: generate_hello.sh
-        content: |
-          #!/bin/sh
-          echo "Hello World" > /sw/hello.txt
-
   - id: batch-job
     source: modules/scheduler/batch-job-template
     use: [network1, appfs]
     settings:
-      runnable: "cat /sw/hello.txt"
+      runnable: |
+        mkdir -p /sw/$BATCH_JOB_UID
+        echo "Hello World from task $BATCH_TASK_INDEX" > /sw/$BATCH_JOB_UID/$BATCH_TASK_INDEX.txt
       machine_type: n2-standard-4
+      task_count: 8
+      task_count_per_node: 4
+      log_policy: "CLOUD_LOGGING"
       instance_image:
         family: batch-centos-7-official
         project: batch-custom-image
+      submit: true
```

Comment:
I suspect this will actually break our integration test that runs for this example as the test submits the job and does not expect it to already be submitted. There are several ways we can address this. Let's discuss after other comments are resolved.

Reply:
A few considerations:
1. This is incompatible with the current integration test. The test will submit the generated job from the login node and then wait for it to successfully conclude and cleans up the job. If we have `submit: true` then, as currently written, there will be no way to clean up the job as we do not know the name.
2. This example is demonstrating the login node use case. The batch job template is loaded onto the login node and instructions are printed for the login node. I think it could be confusing to users if the job is already submitted.

My preference at this time would be to not enable submit on this example and either add it after we have better control of the job name or add a separate example that is focused around this use case, such as with node pools or a builder.

Reply:
Moving forward without submit in the example.